src/components/pages/article/article-page.tsx

```diff
@@ -108,14 +108,32 @@ export const ArticlePage = (props: { metaData: ArticlePageProps, abstract: Conte
           </div>
         </Tab>
         <Tab label="Figures and data">
-          <Heading id="figures" headingLevel={2} content="Figures and data" />
-          <ArticleContent content={getFigures(props.content)} />
+          <JumpToMenu headings={[
```

Comment:
The JumpMenu in the page component need to be built with the headings from the content before we merge this.